### ISSUE TITLE
vomeronasal response

### ISSUE THREAD

**Issue by obophenotype-user** [obophenotype/mammalian-phenotype-ontology#555]
\-------- Original Message --------
Subject: 	MP terms
Date: 	Fri, 19 Feb 2010 11:14:18 -0500
From: 	Melissa Berry &lt;Melissa.Berry@jax.org&gt;
To: 	Cynthia Smith &lt;csmith@informatics.jax.org&gt;

Child of MP:0005423 \(abn somatic nervous system phys\)
Abnormal vomeronasal response to histocompatibility peptides
\(reminder to me, this applies to J:154507\)

This differs from it’s sister avnosmia in that there are responses to 
pheromones, but not to MHC peptides.
There could be a term between MP:0005423 and these sister terms called 
‘abnormal vomeronasal physiology’ but doesn’t seem entirely necessary 
yet with only 2 annotations to these 2 terms.

Also, It seems that avnosmia and anosmia may be too broad and deserving 
of the children:
Specific avnosmia
Specific anosmia
Which represent the loss of the ability to detect particular 
chemosensory stimuli through the \(vomeronasal/olfactory\) system
The one annotation to avnosmia is actually a case of specific avnosmia. 
Perhaps ‘Abnormal vomeronasal response to histocompatibility peptides’ 
is actually couched in ‘specific avnosmia’ so we may not need ‘Abnormal 
vomeronasal response to histocompatibility peptides’ at this point, only 
‘specific avnosmia’.
Your call.  Thanks\!

Reported by: cindyJax

Original Ticket: [obo/mammalian-phenotype-requests/555](https://sourceforge.net/p/obo/mammalian-phenotype-requests/555)

**Comment by cindyJax** [comment]
No, avoiding repsonse terms to specific peptides
